app/scripts/controllers/certificates.js

```diff
@@ -236,4 +247,51 @@ angular.module('openhimWebui2App')
     /****************************************/
 
 
+
+
+    /************************************************/
+    /**         Restart Server Functions           **/
+    /************************************************/
+
+    // server restart later function
+    $scope.restartServerLater = function(){
+      $scope.serverRestartRequired = false;
+    };
+
+    // server restart confirm function
+    $scope.restartServer = function(){
+
+      var restartServer = new Api.Restart();
+      restartServer.$save({}, function(){
+        // restart request sent successfully
+
+        // update restart variables
+        $scope.serverRestarting = true;
+        $scope.serverRestartRequired = false;
+
+        // set estimate time for server restart - 120 seconds
+        $scope.restartTimeout = 10;
```

Comment:
Perhaps we should change this to 5s? It seems to only take like 2-3s for this restart to actually occur.